DataProcessing/datax-host/src/main/scala/datax/processor/CommonProcessorFactory.scala

```diff
@@ -496,8 +496,10 @@ object CommonProcessorFactory {
         postMetrics(Map(s"InputBlobs" -> paths.size.toDouble))
         batchLog.warn(s"InputBlob count=${paths.size}");
 
+        val blobStorageKey = BlobSinker.createBlobStorageKeyBroadcastVariable(paths.head, spark)
```

Comment:
We should move createBlobStorageKeyBroadcastVariable to a separate utility class , feels odd to call blobSinker for read operation in the processor

Reply:
Done